test/external/external_test_yolov8.py

```diff
@@ -67,10 +67,8 @@ def test_forward_pass_torch_onnx(self):
 
     tiny_output = TinyYolov8(input_image)
     
-    # currently rtol is 0.025 because there is a 1-2% difference in our predictions 
-    # because of the zero padding in SPPF module (line 280) maxpooling layers rather than the -infinity in torch. 
-    # This difference does not make a difference "visually". 
-    np.testing.assert_allclose(onnx_output[0], tiny_output.cpu().numpy(), atol=5e-4, rtol=0.025)
+    #minor difference 
+    np.testing.assert_allclose(onnx_output[0], tiny_output.cpu().numpy(),  atol=5e-4,  rtol=0.02)
```

Comment:
like I want to just close this PR due to spacing

Reply:
I guess it requires more work anyway to figure out why there are those differences in the predictions. So closing the PR for now.